composer/workflows/dataflowtemplateoperator_create_dataset_and_table_helper.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is a helper function to the Dataflow Template Operator Tutorial.
+# It helps the user set up a BigQuery dataset and table that is needed
+# for the tutorial.
+
+# [START composer_dataflow_dataset_table_creation]
+
+# Make sure to follow the quickstart setup instructions beforehand.
+# See instructions here:
+# https://cloud.google.com/bigquery/docs/quickstarts/quickstart-client-libraries
+
+# Before running the sample, be sure to install the bigquery library
+# in your local environment by running pip install google.cloud.bigquery
+
+from google.cloud import bigquery
+
+# TODO(developer): Replace with your values
```

Comment:
@tmatsuo I'm personally not a big fan of requiring users to modify the source code to be able to run a sample. Would you be okay with getting these arguments directly from `sys.argv` as positional arguments to avoid using `argparse`?

Maybe something like this?

```
assert len(sys.argv) == 4, 'usage: ataflowtemplateoperator_create_dataset_and_table_helper.py PROJECT LOCATION DATASET_NAME'
project, location, dataset_name = sys.argv[1:]
```

Although `argparse` might be clearer and would have better error reporting. What do you think?

Reply:
It's against our Code Snippet Style Guide: go/code-snippets-style#running-from-the-command-line

Reply:
@davidcavazos 
Personally I have a slight preference on using `sys.argv`, but I also understand the rationale of the style guide. The rationale of discouraging the use of `sys.argv` or `argparse` is value/maintenance cost ratio.

Our assumption: majority of users stay on the cloudsite docs and copy the code into their project to start. Providing a CLI only add values for small amount of users who actually git clone the sample repo.

There is certain amount of maintenance cost for providing a CLI. The current main is also not tested, but it's rather easy to maintain.

Does it make sense?